repos/fdbt-site/src/pages/api/editCarnetProperties.ts

```diff
@@ -0,0 +1,81 @@
+import { ErrorInfo, NextApiRequestWithSession } from '../../interfaces';
+import { NextApiResponse } from 'next';
+import { getSessionAttribute, updateSessionAttribute } from '../../utils/sessions';
+import {
+    EDIT_CARNET_PROPERTIES_ERROR,
+    MATCHING_JSON_ATTRIBUTE,
+    MATCHING_JSON_META_DATA_ATTRIBUTE,
+} from '../../constants/attributes';
+import { redirectTo, redirectToError } from '../../utils/apiUtils';
+import { putUserDataInProductsBucketWithFilePath } from '../../utils/apiUtils/userData';
+import { isValidInputDuration, removeExcessWhiteSpace, checkIntegerIsValid } from '../../utils/apiUtils/validator';
+
+const validateDuration = (carnetDuration: string, carnetDurationUnit: string, quantity: string): ErrorInfo[] => {
```

Comment:
This function either needs to be exported, and tested in isolation, or tested via supplying the API different inputs and making sure it picks up on them.

editPeriodDuration.ts (validateDuration function) has the same issue, which also needs addressing please.

Reply:
It is reused from multipleProducts but will do some tests for this

Reply:
If its reused, then we can just import the direct function. If its new, it needs testing just the same.